src/net.h

```diff
@@ -31,6 +31,7 @@
 
 #ifndef WIN32
 #include <arpa/inet.h>
+#include <poll.h>
```

Comment:
It doesn't seem like the current code is using `poll` on Windows.

Reply:
it isnt?